mis_builder/models/mis_report_instance_annotation.py

```diff
@@ -0,0 +1,71 @@
+# Copyright 2025 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class MisReportInstanceAnnotation(models.Model):
+    _name = "mis.report.instance.annotation"
+    _description = "Mis Report Instance Annotation"
+
+    period_id = fields.Many2one(
+        comodel_name="mis.report.instance.period",
+    )
+    kpi_id = fields.Many2one(
+        comodel_name="mis.report.kpi",
+    )
+    subkpi_id = fields.Many2one(
+        comodel_name="mis.report.subkpi",
+    )
```

Comment:
`ondelete="cascade"` on period_id, kpi_id, subkpi_id ?

Some `required=True`?

Also add a composite index on `period_id, kpi_id, subkpi_id` to anticipate when many old notes accumulate?

Reply:
done :+1: